db/models/categories.model.js

```diff
@@ -19,4 +19,35 @@ exports.selectReviewById = (review_id) => {
         });
     }
 
+exports.selectCommentByReviewId = (review_id) => {
+        const query = `
+          SELECT * 
+          FROM comments 
+          WHERE review_id = $1
+          ORDER BY created_at DESC
+        `;
+        const values = [review_id];      
+        return connection
+        .query(query, values)
+        .then((result) => {
+          const comments = result.rows;
+    
+          if (comments.length === 0) {
```

Comment:
The error for an invalid review id should be thrown before making a query for the comments